app/views/optIn/ConfirmTaxYear.scala.html

```diff
@@ -0,0 +1,70 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
```

Comment:
this file could be named 'OptInCompletedSingleYearView' inline with existing file name